File: C/1045.c
#include <stdio.h>

int main() {
    float A, B, C, AUX;
    scanf("%f %f %f", &A, &B, &C);

    if (A < B){
      AUX = A;
      A = B;
      B = AUX;
    }
    if (A < C){
      AUX = A;
      A = C;
      C = AUX;
    }
    if(B < C){
      AUX = B;
      B = C;
      C = AUX;
    }

    
    
    if (A >= B + C) {
        printf("NAO FORMA TRIANGULO\n");
    }
   else {
        if (A * A == B * B + C * C) {
            printf("TRIANGULO RETANGULO\n");
        }
        if (A * A > B * B + C * C) {
            printf("TRIANGULO OBTUSANGULO\n");
        }
        if (A * A < B * B + C * C) {
            printf("TRIANGULO ACUTANGULO\n");
        }
        if (A == B && B == C) {
            printf("TRIANGULO EQUILATERO\n");
        }
        if ((A == B && A != C) || (A == C && A != B) || (B == C && B != A)) {
            printf("TRIANGULO ISOSCELES\n");
        }
    }
    return 0;
}

File: Python/1045.py
vetor = []
A, B, C = input().split()
vetor.append(float(A))
vetor.append(float(B))
vetor.append(float(C))

vetor.sort(reverse=True)
A, B, C = vetor
def function(A, B, C):
    if A == B == C:
        print("TRIANGULO EQUILATERO")
    elif A == B != C or A == C != B or B == C != A:
        print("TRIANGULO ISOSCELES")


if A>= (B + C):
    print("NAO FORMA TRIANGULO")
elif A**2 == (B**2 + C**2):
    print("TRIANGULO RETANGULO")
elif A**2 > (B**2 + C**2):
    print("TRIANGULO OBTUSANGULO")
    function(A, B, C)
elif A**2 < (B**2 + C**2):
    print("TRIANGULO ACUTANGULO")
    function(A, B, C)
